paasta_tools/cli/cmds/check.py

```diff
@@ -322,21 +333,22 @@ def smartstack_check(service, service_path):
 def paasta_check(args):
     """Analyze the service in the PWD to determine if it is paasta ready
     :param args: argparse.Namespace obj created from sys.args by cli"""
-    service = figure_out_service_name(args)
-    service_path = os.path.join('/nail/etc/services', service)
+    service_root = args.yelpsoa_config_root
```

Comment:
Instead of calling this "service_root" everywhere, can you call it just call soa_dir?